project.mml

```diff
@@ -2102,13 +2119,15 @@ Layer:
             access,
             CONCAT(
                 name,
-                CASE WHEN name IS NOT NULL AND elevation IS NOT NULL THEN E'\n' ELSE NULL END,
-                CASE WHEN elevation IS NOT NULL THEN CONCAT(REPLACE(ROUND(elevation)::TEXT, '-', U&'\2212'), U&'\00A0', 'm') ELSE NULL END
+                CASE WHEN elevation IS NOT NULL THEN CONCAT(E'\n', REPLACE(ROUND(elevation)::TEXT, '-', U&'\2212'), U&'\00A0', 'm') ELSE NULL END,
```

Comment:
I made a mistake here.

This is the code that appends the elevation text to the name column. It does need to be modified by this PR because now both elevation and height are rendered, whereas before it was only elevation.

I tried to simplify the expression by forcing a newline before the elevation/height text, but that doesn't work when the name is null. I have put it back to the original structure.

Reply:
I meant only this line of code, but now it's the same as before, so no problem here.